extensions/word-search/src/search-results.tsx

```diff
@@ -35,26 +37,26 @@ export default function SearchResults(
           description={search !== "" ? "No Results Found" : helperDescription}
         />
       ) : (
-        words.map((word) => (
-          <List.Item
-            icon={"command-icon.png"}
-            key={word.word}
-            title={word.word}
-            subtitle={word.defs !== undefined ? word.defs[0] : ""}
-            actions={<Actions {...{ word, search }} defaultAction={defaultAction} />}
-          />
-        ))
+        words.map((word) => {
+          const wordString = searchIsCapitalized ? word.word[0] + word.word.slice(1) : word.word;
```

Comment:
This logic is not capitalizing the word.

Try this instead:

    const wordString = searchIsCapitalized ? word.word.charAt(0).toUpperCase() + word.word.slice(1) : word.word;

Reply:
```suggestion
          const wordString = searchIsCapitalized ? word.word.charAt(0).toUpperCase() + word.word.slice(1) : word.word;
```